### PR TITLE
fix: LSDV-E-135: Manual security updates

### PR DESCRIPTION
**Pull request by Travis1282**
### PR fulfills these requirements
- [ ] Commit message(s) and PR title follows the format `[fix|feat|ci|chore|doc]: TICKET-ID: Short description of change made` ex. `fix: DEV-XXXX: Removed inconsistent code usage causing intermittent errors`
- [ ] Tests for the changes have been added/updated (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] Best efforts were made to ensure docs/code are concise and coherent (checked for spelling/grammatical errors, commented out code, debug logs etc.)
- [ ] Self-reviewed and ran all changes on a local instance (for bug fixes/features)

#### Change has impacts in these area(s)
_(check all that apply)_
- [ ] Product design
- [ ] Frontend

### Describe the reason for change
_(link to issue, supportive screenshots etc.)_

#### What does this fix?
_(if this is a bug fix)_

#### What is the new behavior?
_(if this is a breaking or feature change)_

#### What is the current behavior?
_(if this is a breaking or feature change)_

#### What libraries were added/updated?
_(list all with version changes)_

#### Does this change affect performance?
_(if so describe the impacts positive or negative)_

#### Does this change affect security?
_(if so describe the impacts positive or negative)_

#### What alternative approaches were there?
_(briefly list any if applicable)_

#### What feature flags were used to cover this change?
_(briefly list any if applicable)_

### Does this PR introduce a breaking change?
_(check only one)_
- [ ] Yes, and covered entirely by feature flag(s)
- [ ] Yes, and covered partially by feature flag(s)
- [ ] No
- [ ] Not sure (briefly explain the situation below)

### What level of testing was included in the change?
_(check all that apply)_
- [ ] e2e
- [ ] integration
- [ ] unit

### Which logical domain(s) does this change affect?
_(for bug fixes/features, be as precise as possible. ex. Authentication, Annotation History, Review Stream etc.)_

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -197,7 +197,7 @@
     "webpack": "^5.79.0",
     "webpack-cli": "^5.0.1",
     "webpack-dev-server": "^4.13.3",
-    "xml2js": "^0.4.23",
+    "xml2js": "0.6.x",
```
Comment (hlomzik) [inline package.json:200]: should not it be `^0.6.0`?

File: e2e/package.json [hunk 2]
```diff
@@ -36,7 +36,7 @@
     "eslint": "^8.4.1",
     "eslint-plugin-codeceptjs": "^1.3.0",
     "mkdirp": "^1.0.4",
-    "node-fetch": "^2.6.1",
+    "node-fetch": "^2.6.7",
```
Comment (hlomzik) [inline e2e/package.json:39]: `fetch` is not used here as well. please, remove it

File: package.json [hunk 3]
```diff
@@ -131,7 +131,7 @@
     "antd": "^4.3.3",
     "autoprefixer": "^10.4.2",
     "babel-jest": "^29.3.1",
-    "babel-loader": "^8.2.2",
+    "babel-loader": "8.2.2",
```
Comment (hlomzik) [inline package.json:134]: do we really need to make version fixed here?

**Review by hlomzik — CHANGES_REQUESTED** [review]

**Review by hlomzik — CHANGES_REQUESTED** [review]

File: package.json [hunk 4]
```diff
@@ -197,9 +197,12 @@
     "webpack": "^5.79.0",
     "webpack-cli": "^5.0.1",
     "webpack-dev-server": "^4.13.3",
-    "xml2js": "^0.4.23",
+    "xml2js": "0.6.x",
     "xpath-range": "^1.1.1"
   },
+  "resolutions": {
+    "loader-utils": "1.4.2"
+  },
```
[the remark this answers is not in the document]
Comment (Travis1282) [reply package.json:205]: yea dependabot wants this version. so we should make the one in dm match? 
https://github.com/heartexlabs/label-studio-frontend/security/dependabot/84

File: package.json [hunk 5]
```diff
@@ -197,9 +197,12 @@
     "webpack": "^5.79.0",
     "webpack-cli": "^5.0.1",
     "webpack-dev-server": "^4.13.3",
-    "xml2js": "^0.4.23",
+    "xml2js": "0.6.x",
```
Comment (bmartel) [inline package.json]:
```suggestion
    "xml2js": "^0.6.0",
```

**Review by bmartel — APPROVED** [review]

**Pull request closed without merging**